docs/CHANGES.md

```diff
@@ -0,0 +1,59 @@
+# Changes
+
+This document is a place to track all fo the changes that we
+believe must be addressed in other sections of the OCI in
+order to enable the efforts of the WG.
+
+## Distribution Spec
+
+The following is a list of changes may need to be
+addressed in [OCI Distribution Spec](https://github.com/opencontainers/distribution-spec)
+in order to support the goals of this WG:
+
+### Pull
+
+- [add item here]
+
+### Push
+
+- [add item here]
+
+### Content Discovery
+
+- [add item here]
+
+### Content Management
+
+- [add item here]
+
+### Uncategorized
+
+- [add item here]
+
+## Image Spec
```

Comment:
There is a call out that image-spec that specifically doesn't call out distribution and since this aligns with distribution, we should be careful to disambiguate that this is not a runtime spec and any changes to image spec might have a cascading effect. 

Image spec verbiage - `Q: Why doesn't this project mention distribution?`

Reply:
Some docs on the topic of what/where/how:
- [Decouple the Persistence Spec from the Specific Artifact Spec](https://hackmd.io/iOemxuZaT8O5WciRLm0K9g)
- [Discussion of a new manifest](https://hackmd.io/9cUQf-dYR-WUzcELG6RoXQ)
    This one is a little more assumptive of an outcome, but may be an interesting read with new insight to the scenarios